admin/herbert_mex_mpi.m

```diff
@@ -12,13 +12,14 @@
 input_files = cellfun(@(fn)fullfile(code_folder,fn),input_files,'UniformOutput',false);
 
 add_include = ['-I',mpich_folder,'include'];
-add_lib = ['-L',mpich_folder,'lib'];
+mpich_lib = fullfile(mpich_folder,'lib');
```

Comment:
defining this as '/usr/local/mpich/lib' does this work on Windows?

Reply:
Have doubts if this script is appropriate at all in favour of cmake. But after morinign discussion will change it properly.